packages/xo-server-auth-github/src/index.js

```diff
@@ -23,8 +23,13 @@ class AuthGitHubXoPlugin {
     this._xo = xo
   }
 
-  configure(conf) {
+  async configure(conf, { loaded }) {
     this._conf = conf
+
+    if (loaded) {
```

Comment:
Should it be included in the plugin life cycle implementation?

Reply:
It could, but not by default, this is not necessary for all plugins.

Reply:
Independent of the plugin implementation and need, we can unload/load the plugin to be sure that it's load implementation takes care of the new changes.

Reply:
I don't think we should do it automatically if that's not required.